ui/plugins/pinboard.js

```diff
@@ -126,6 +126,12 @@ treeherder.controller('PinboardCtrl', [
         };
 
         $scope.canSaveClassifications = function() {
+            if ($scope.enteringBugNumber) {
+                // we should save this for the user, as they likely
+                // just forgot to hit enter.
+                $scope.saveEnteredBugNumber();
+            }
+            $scope.completeClassification();
```

Comment:
So does saveEnteredBugNumber(), to be fair. Thought I'd err on the side of making sure canSaveClassifications() succeeds also in the case where the classification comment section is still being edited. 

I can probably drop this, though, since it was working just fine since it's just a textbox vs the bug number's magic stuff.